tests/robustness/validate/validate_test.go

```diff
@@ -493,7 +530,37 @@ func TestValidateWatch(t *testing.T) {
 				putPersistedEvent("b", "2", 3, true),
 				putPersistedEvent("c", "3", 4, true),
 			},
-			expectError: errBrokeBookmarkable.Error(),
+			expectError: errBrokeReliable.Error(),
+		},
+		{
+			name: "Resumable, Reliable, Bookmarkable - empty events between progress notifies - fail",
```

Comment:
can you add a test case for `Key: "d"` with everything else the same to test empty events between progress notifies for unmatched key - pass ?

Reply:
Done